noir-projects/aztec-nr/aztec/src/oracle/get_public_data_witness.nr

```diff
@@ -16,11 +16,11 @@ struct PublicDataWitness {
 #[oracle(getPublicDataTreeWitness)]
 unconstrained fn get_public_data_witness_oracle(
     _block_number: u32,
-    _leaf_slot: Field
+    _storage_index: Field
```

Comment:
Similar to a comment above, perhaps we should just call this 'public data tree index'. I do agree however that 'leaf slot' is confusing here.

Reply:
I checked the implementation of `getPublicDataTreeWitness` and indeed in this case it represents the actual index in the tree. Agree that public data tree index is the clearest option of all.

But if we rename it to that then I feel contract_storage_index is kinda 💩 Will try to think of something better for that.

Would address the naming issues in https://github.com/AztecProtocol/aztec-packages/issues/7551